### DISCUSSION TITLE
Excellent System

### DISCUSSION THREAD

**Discussion by ceewanna** [roflcoopter/viseron#632] [General]
I have been exploring viseron setup over these few days. Today, I just figure out how to set up the built-in webserver (my existing system seems to have taken up 8888 and I am new to docker so I have to map it to another number.)

After seeing the webpage, together with the test system I put on testing over the weekend, I feel so grateful to the developer. I have been using commercial nvr and another system running along with the nvr. That system doesn't record audio and has very limited user interface, particularly the ability to view past recordings. However, it is a stable and easy to setup system consuming reasonable cpu workload.

Right now I am still trying to understand and finalize all the setup parameters and tuning. Also if viseron can provide 24/7 recording in parallel to the motion detection, that would be even better. Hopefully I could eventually replace my existing system with viseron if I could understand all the settings.

**Comment by roflcoopter** [comment]
I'm glad you like it!
If you have any questions, dont hesitate to ask them. I will try to answer as fast as possible.

**Comment by ceewanna** [comment]
Thank you so much.
